aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -257,6 +257,7 @@ Resources:
 <% if frontends-%>
 <%=  component 'cache'%>
 <% end -%>
+<%= component 's3_backed_subdomains', bucket_name: 'cdo-lesson-plans', subdomain: 'lesson-plans' -%>
```

Comment:
I think we’ll need to only invoke this component for the production environment:

```
<% if rack_env? :production-%>
  <%= component ‘s3_backed_subdomains’ -%>
<% end -%>
```

`cloud_formation_stack.yml.erb` is used to generate a CloudFormation Template for each of our managed environments (`staging`, `test`, `levelbuilder`, `production`, and each adhoc as well). My understanding is that we currently have a single `tts` subdomain that all managed environments and even development environments share and that by committing “levelbuilder” content (`.level` files, etc.) into our source control system, we control which environments use which content served up by the `tts` subdomain and that we’ll have a single `lesson-plan` subdomain that’s shared by all environments with versioning similarly controlled indirectly by choosing which URLs levelbuilder content points to within this shared subdomain.

Reply:
oh, good call!